mmv1/products/gkeonprem/VmwareCluster.yaml

```diff
@@ -0,0 +1,662 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: "VmwareCluster"
+min_version: beta
+base_url: "projects/{{project}}/locations/{{location}}/vmwareClusters"
+create_url: "projects/{{project}}/locations/{{location}}/vmwareClusters?vmware_cluster_id={{name}}"
+update_url: "projects/{{project}}/locations/{{location}}/vmwareClusters/{{name}}"
```

Comment:
We'll need to add update test(s) as well as the current single-step example tests: https://googlecloudplatform.github.io/magic-modules/docs/how-to/add-mmv1-test/#update-tests

We'll want to test every mutable field we can- any fields not marked `output: true` or `immutable: true`.

Reply:
update tests are added now covering all mutable fields. Thanks for the guidance!